### DISCUSSION TITLE
deny updating a hidden ref

### DISCUSSION THREAD

**Discussion by tonyzhao-jt** [community/community#110485] [New to GitHub]
Hi, i need to remove some sensitivity data from my repo, and use [BFG](https://docs.github.com/en/enterprise-cloud@latest/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository) for this for the moment, but now I encountered 
```
 ! [remote rejected] refs/pull/1/head -> refs/pull/1/head (deny updating a hidden ref)
```
Anyone knows how to get some support from the Github to help me erase them?

**Comment by tonyzhao-jt** [comment]
I just found it could be done here: 
https://support.github.com/request/remove-data?
Thanks at all.

**Comment by lacy-rvo** [comment]
https://github.com/rtyley/bfg-repo-cleaner/issues/36#issuecomment-37877829
